app/waf/src/controllers/application.controller.ts

```diff
@@ -257,24 +257,23 @@ export class ApplicationController {
       );
 
       if (params.endpointpolicy) {
-        params.rules = await this.ruleRepository.find({
-          where: {
-            id: {
-              inq: (<Endpointpolicy>params.endpointpolicy).rules,
-            },
-          },
-        });
+        let eppolicy = <Endpointpolicy>params.endpointpolicy;
+        params.rules = await this.endpointpolicyRepository
+          .rules(eppolicy.id)
+          .find();
+
         let rules = <Rule[]>params.rules;
         for (let rule of rules) {
           rule.conditions = await this.ruleRepository
             .conditions(rule.id)
             .find();
           rule.actions = await this.ruleRepository.actions(rule.id).find();
         }
-        params.wafs = await this.wafpolicyRepository.find();
       }
     }
 
+    params.wafs = await this.wafpolicyRepository.find();
```

Comment:
Will we attach all wafpolicy to the deployment without filter condition?

Reply:
for this deploy functionality . we could leave it alone.